cookiecutter/bootstrap/{{ cookiecutter.__project_slug }}/pyproject.toml

```diff
@@ -0,0 +1,139 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "{{ cookiecutter.__project_slug }}"
+version = "0.0.1.dev"
+description = "Command line interface to talk to pulpcore's REST API. ({{ cookiecutter.app_label | capitalize }} plugin commands)"
+readme = "README.md"
+requires-python = ">=3.6"
+license = {text = "GPLv2+"}
+authors = [
+  {name = "Pulp Team", email = "pulp-list@redhat.com"},
+]
+classifiers=[
+  "Development Status :: 4 - Beta",
+  "Environment :: Console",
+  "Intended Audience :: System Administrators",
+  "License :: OSI Approved :: GNU General Public License v2 or later (GPLv2+)",
+  "Operating System :: OS Independent",
+  "Programming Language :: Python :: 3",
+  "Topic :: System :: Software Distribution",
+  "Typing :: Typed",
+]
+dependencies = [
+  "pulp-cli>=0.23.1,<0.24.0.dev",
+  "pulp-glue-{{ cookiecutter.app_label }}==0.0.1.dev",
+]
+
+[project.urls]
+repository = "https://github.com/pulp/{{ cookiecutter.__project_slug }}"
+changelog = "https://github.com/pulp/{{ cookiecutter.__project_slug }}/blob/main/CHANGES.md"
+
+[project.entry-points."pulp_cli.plugins"]
+{{ cookiecutter.app_label }} = "pulpcore.cli.{{ cookiecutter.app_label }}"
+
+[tool.setuptools.packages.find]
+where = ["."]
+include = ["pulpcore.cli.*"]
+namespaces = true
+
+[tool.setuptools.package-data]
+"*" = ["py.typed", "locale/*/LC_MESSAGES/*.mo"]
+
+[tool.pulp_cli_template]
+app_label = "{{ cookiecutter.app_label }}"
+glue = true
+docs = false
+translations = false
+test_matrix = """
+- python: "3.11"
+  image_tag: "nightly"
+  pulp_api_root: "/relocated/djnd/"
+- python: "3.6"
+  image_tag: "latest"
+  lower_bounds: true
+"""
+
+[tool.towncrier]
+filename = "CHANGES.md"
+directory = "CHANGES/"
+title_format = "## {version} ({project_date})"
+template = "CHANGES/.TEMPLATE.md"
+issue_format = "[#{issue}](https://github.com/pulp/{{ cookiecutter.__project_slug }}/issues/{issue})"
+start_string = "[//]: # (towncrier release notes start)\n"
+underlines = ["", "", ""]
+
+[[tool.towncrier.section]]
+path = ""
+name = ""
```

Comment:
Are these suppose to be blank?

Reply:
yes, that's the main section in the changelog.